autoload/lsp.vim

```diff
@@ -695,6 +695,8 @@ function! s:on_notification(server_name, id, data, event) abort
                 call lsp#ui#vim#diagnostics#handle_text_document_publish_diagnostics(a:server_name, a:data)
             elseif l:response['method'] ==# 'textDocument/semanticHighlighting'
                 call lsp#ui#vim#semantic#handle_semantic(a:server_name, a:data)
+            elseif l:response['method'] ==# 'window/logMessage'
```

Comment:
Yeah, I'm fine not including the `logMessage` change. I think it would be nice to be able to catch these somehow, but I agree that using `echo` is probably too intrusive. I included it primarily to help debug `showMessageRequest`, but I can remove it from this PR.

Reply:
I just updated the PR to make `lsp#ui#vim#window#log_message` a noop, so that it can be updated in the future to save that message to a logfile, or some other option that's less intrusive.